### DISCUSSION TITLE
Can I draw with a finger?

### DISCUSSION THREAD

**Discussion by eownerdead** [flxzt/rnote#1084] [Q&A] [answered]
I'm using rnote 0.8.2 and want to write with a finger. However, when I try to write, the page still scrolls, though I changed some settings randomly. Thanks.

```
$ uname -a
Linux slate 6.1.84 #1-NixOS SMP PREEMPT_DYNAMIC Wed Apr  3 13:19:55 UTC 2024 x86_64 GNU/Linux
```

**Comment by flxzt** [comment, marked as answer]
Yes, you should be able to toggle drawing with touch through this menu item:

![Bildschirmfoto vom 2024-04-21 15-54-17](https://github.com/flxzt/rnote/assets/19841886/6725b92d-5c68-44aa-baca-50688454b9f4)

**Reply by eownerdead** [reply]
How come I could not find this? Thanks!
